modules/artifact/src/main/java/org/mule/runtime/module/artifact/classloader/ClassLoaderFilter.java

```diff
@@ -7,10 +7,12 @@
 
 package org.mule.runtime.module.artifact.classloader;
 
+import java.io.Serializable;
+
 /**
  * Determines if a given class or resource is exported in a plugin classloader
  */
-public interface ClassLoaderFilter
+public interface ClassLoaderFilter extends Serializable
```

Comment:
Not need to be, it was needed while doing different implementations of the junit runner. This change was reverted.